.github/workflows/edFi.common pullrequest.yml

```diff
@@ -0,0 +1,29 @@
+name: EdFi.Common Pull request build and test
+
+on:
+  pull_request:
+    branches: [main]
+
+env:
+  INFORMATIONAL_VERSION: "5.4"
+  CONFIGURATION: "Release"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
```

Comment:
It appears this one still references v1, and so does `edfi.ods.api pullrequest.yml` so could you please update both of these to reference the v2 hash?

Reply:
I fixed it for all builds